packages/leva/src/useControls.ts

```diff
@@ -9,21 +9,24 @@ import shallow from 'zustand/shallow'
 type HookSettings = { store?: StoreType }
 type SchemaOrFn<S extends Schema = Schema> = S | (() => S)
 
-type FunctionReturnType<S extends Schema> = [
+type FunctionReturnType<S extends Schema, FOLDER extends string | void> = [
```

Comment:
`FOLDER` is not being used elsewhere... Also types are just capitalized not uppercased here :)